doc/classes/PCKPacker.xml

```diff
@@ -15,6 +15,14 @@
 			<argument index="1" name="source_path" type="String">
 			</argument>
 			<description>
+				The [PCKPacker] is used to create packages in application runtime.
+				[codeblock]
+				var Packer = new PCKPacker();
+				Packer.PckStart("Test.pck", 0)
+				Packer.AddFile("res://Text.txt", "Test.txt");
+				Packer.Flush(false);
```

Comment:
The methods should be written in snake_case.

Reply:
Sorry, I could not figure out how to make a tab-like-thing that allows switching in-between languages, I am going to port this code to gd then.

Reply:
Language tabs aren't available in the internal docs.